doc/whats_new.rst

```diff
@@ -157,9 +157,15 @@ BUG
     - Fix bug in :func:`mne.viz.plot_compare_evokeds` where ``truncate_yaxis`` was ignored (default is now ``False``), by `Jona Sassenhagen`_
 
     - Fix bug in :func:`mne.viz.plot_evoked` where all xlabels were removed when using ``spatial_colors=True``, by `Jesper Duemose Nielsen`_
+<<<<<<< HEAD
```

Comment:
This (and other in whats_new) is from unresolved merge conflict, you should clean it up